File: node-ts/BillsApi/src/application/bills/create/CreateBillCommand.ts
export type CreateBillLineCommand = {
  concept: string;
  quantity: number;
  unitAmount: number;
};

export type CreateBillCommand = {
  billNumber: string;
  issuedAt: string;
  customerName: string;
  currency: string;
  tax: number;
  lines: CreateBillLineCommand[];
};

File: kotlin/BillsApi/src/main/kotlin/com/apilangarena/billsapi/application/bills/create/CreateBillCommand.kt
package com.apilangarena.billsapi.application.bills.create

import java.math.BigDecimal
import java.time.LocalDate

data class CreateBillLineCommand(
    val concept: String,
    val quantity: BigDecimal,
    val unitAmount: BigDecimal,
)

data class CreateBillCommand(
    val billNumber: String,
    val issuedAt: LocalDate,
    val customerName: String,
    val currency: String,
    val tax: BigDecimal,
    val lines: List<CreateBillLineCommand>,
)
